tests/foreman/api/test_docker.py

```diff
@@ -1524,24 +1540,39 @@ def test_update_registry_url(self):
 
         @Assert: the external registry is updated with the new URL
 
-        @Status: Manual
-
         """
+        url = gen_url(subdomain=gen_string('alpha'))
+        new_url = gen_url(subdomain=gen_string('alpha'))
+        registry = entities.Registry(
+            name=gen_string('alpha'),
+            url=url,
+        ).create()
+        self.assertEqual(registry.url, url)
+        registry.url = new_url
+        registry = registry.update()
+        self.assertEqual(registry.url, new_url)
```

Comment:
Decided to leave code as is for better readability. So please proceed with review